modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -152,12 +121,6 @@ export const initAndShow = (editor: Editor, e: EditorEvent<TouchEvent>, buildMen
       editor.once('touchend', () => editor.off('selectionchange', openiOS));
     }
   } else {
-    // On Android editor.selection hasn't updated yet at this point, so need to do it manually
-    // Without this longpress causes drag-n-drop duplication of code on Android
-    if (isAndroid && !useNodeAnchor) {
-      editor.selection.setCursorLocation(e.target, 0);
-    }
-
```

Comment:
Removed this part cause it was removing the selection and moving the cursor to the beginning of the node when longpressing on Android.

Reply:
This can't be removed as it was needed on older versions of Android. We'll need a way/some heuristic to determine when it's needed.